### DISCUSSION TITLE
考試週vs.考試月

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1769] [General]
再過不久就要迎來期末考了，而我的科系是一次考很多科的，所以每到期末考週，腦袋就是狂轟亂炸、睡眠時間也是直接消失，深陷熬夜讀書的漩渦。

所以比起期末考週，我更喜歡'考試月'，就是每個禮拜都有考試，但只有一兩科，雖然可能每天都會感到有點煩躁，但壓力會小許多，可以放寬心態去準備。

在這邊也問問看大家的想法，到底是喜歡期末考週呢還是期末考月？

**Comment by Ryanlolry** [comment]
我認為各有優缺吧，我們的科系在二上各科老師有商量過，讓考試平均分配在各週，導致有兩個月左右的時間中，我們每週都有考試，且期末考週一週有三科，所以我認為對我來說，每週考試相對好一點的，讓我們有多一點時間去準備，但我認為仍有點問題，我的習慣是如果當週要考試我就會幾乎花所有時間去讀他，所以會造成其他進度落後，導致他要考的時候花更多時間去準備他。

**Comment by v0ouo** [comment]
我自己是比較喜歡"考試月"，如同作者所說的，我認為如果所有考試都全部放在同一或兩個禮拜，那真的會吃不消，而且壓力會非常的大。如果一個禮拜只有一兩科考試要考的話，那壓力真的會小不少，甚至還會有不少時間是可以做自己想做的事。

**Comment by mrweixd** [comment]
我超喜歡考試週的，全部東西擠在一起的感覺超刺激的。而且這樣忙完才有解脫的感覺。

**Comment by Yanxxx7979** [comment]
兩個都體驗過:) 曾經過過考試學期每周都在考試，會很躁但能夠好好準備，可能出現的問題是彈性疲乏後不想讀
考試週就是一次炸裂，根本讀不完會想要直接去世，但考完就是海闊天空

**Comment by sorn32** [comment]
兩種都有經歷過，我還是比較喜歡考試週，雖然炸了就完蛋，但考完的那種爽感比考試月的溫水煮青蛙還要好ヾ(*´∀ ˋ*)ﾉ

**Comment by vchien03** [comment]
考試月會給我一種分散風險的感覺，不會每科都撞在一起，壓力比較小。
但就是沒有考試週那種「考完即解脫」的爽感。

**Comment by chuanmin0317** [comment]
雖然考試週結束後，就可以完全釋放壓力，不需要長時間處於考試壓力之下，不過面對多科目的同時考試，會讓我在短時間感到更大的壓力，所以我覺得自己比較喜歡考試月，而且這樣我有比較多時間可以準備單一的科目，取得更好的成績

**Comment by FakeWood** [comment]
大一下的時候，微積分分三次考，再加上系上有一些神奇考試時間，就導致從期中到期末一直都在考試，
我覺得頗累的，都沒有能喘口氣的時間，
所以我覺得期末考集中一點比較好，
再說，就算期中考都集中在同一周，也可以提前一個月準備，
這樣意思就跟考試月一樣了。

**Comment by lyykey** [comment]
兩者比較下，我更愛考試周，比起備考是轟炸，我更愛一步一步的完成每個挑戰，密集的考試攻擊其實帶給我們更多的是及格邊緣，甚至被當的危險，以我過去的經驗，考試周也比考試月帶來的學習成效更大，能夠記住的東西也比較多。

**Comment by benbebobo149** [comment]
我比較傾向於讓考試比較分散，因為這會讓我有更多時間準備，也可以更好地吸收內容。或者以專題形式取代考試，我認為也是一個不錯的想法，因為在研究某個議題時，我會更深入地了解該科目的細節，從中學到更多知識。這樣的學習方式可以激發我更大的興趣，促使我主動地探索知識的深度和廣度。我也認為這樣的方式更能培養我們的批判性思維和解決問題的能力，因為在專題研究中，我們需要分析、評估和提出解決方案。最後，我要感謝同學的分享，這篇文章讓我深入思考了不同的學習方式，對我很有啟發性。

**Comment by willychiuchiu** [comment]
理想上喜歡考試週，就可以不用長時間都處在壓力下，對自己身心靈上的發展都會比較好。但現實上因為考試科目實在太多，若全部都集中在同一週中，我覺得自己完全沒有辦法應付，就只是把考試考完而已，用的可能是死背硬記，並沒有辦法完全吸收，因此我認為考試分散還是會比較好。

**Comment by 110101501** [comment]
自己系上的考試主要是以考試月為主，考試的時程基本上都很分散。好處是每次的考試科目不會過於集中，準備起來時間較為充裕；壞處是沒有閒下來休息的時間，考完一科後就要開始準備下一科，甚至是自期中考後就會一路考到期末。
每當聽到其他學校的朋友是考試週，當週考完當週解脫，不禁都會有點小羨慕。但自從聽到他們提及說他們當週都要考很多科，壓力也很大沒辦法休息，就會開始覺得考試月好像也沒有當初想得那麼糟了。

**Comment by YuanWang08** [comment]
我喜歡考試周，對於我個人而言，念一個技術含量比較低的學科段考大概只要一兩天就可以拿到及格的分數，若是將時間拖到一個月就有點太長了，反而消磨了想要奮發學習的拚勁。我覺得考試週與考試月因人而異，主要還是受個人學習習慣的影響。

**Comment by jeff110303549** [comment]
我也覺得考試週非常摧殘學生，即便平時課堂也沒有過多打混摸魚，不過每到期中期末還是有很多知識要點需要時間和精力鞏固後才能為己所用，考試週這樣的方式雖然讓學生痛苦的時間縮短了些，但也隱性地在短時間內讓學生的壓力暴增(我自己就因為考試週爆熬夜也長了兩次泡疹)

**Comment by YANRONG0123** [comment]
在我的系上，通常期中考是考試月，因為有分三次考試，也有兩次考試，在1/3的考試結束後，接著是1/2的考試，再來就是2/3的考試，而期末考就是集中在兩週進行，對我來說，只要平常上課認真，回家有稍微複習一下，應該都可以低空飛過那些困難的科目。

**Comment by LinChloe** [comment]
喜歡考試月，全部都同一周太可怕了。
我是考試考太多腦袋會有點混亂的類型。
尤其有些科目要背東西(考古)，上一科考完後我腦袋會呈現聖人狀態，當天再繼續考是會很慌張，什麼都想不起來。
真的很好奇我大學前怎麼度過來的。

**Comment by thomasxshen** [comment]
我覺得考試週和考試月各有其優缺點，這樣的安排可以讓學生在短時間內集中精力準備考試，但也可能會造成相當大的壓力。而考試月則是指在一個月內分散安排考試，這可能會給學生更多的時間準備每一科目，但也可能導致考試期間的壓力和焦慮持續更長時間。不同的學校和教育系統會根據他們的特定需求選擇不同的安排。

**Comment by yxin214** [comment]
我喜歡考試月，雖然考試月沒辦法給人一種真正休息的感覺(放很常一段時間)，但相較考試週的壓力並不那麼大，而且每週一兩科考完便能好好休息個一兩天再繼續，不會像考試週一樣連環考連個喘息時間都沒有。

**Comment by wanka2030** [comment]
高中的時候，學校會利用兩三天來檢測學生平時上課的實力，也讓每位老師都充滿進度壓力、必須在時間內將不同量的課程內容一次教完。但大學之後，雖然考試的時間都會間隔了一、兩個禮拜，但也能讓學生好好學習複習，這樣也能夠讓自己在不斷考試的壓力之中稍喘一口氣
!😊

**Comment by YifenLin** [comment]
我也是喜歡"考試月"派，這樣各科都有足夠的時間準備，對我言比較好吸收。我自認我是一個坐不住的人，如果是"考試週"就必須一整天坐著讀書，我會坐不住，所以遇到考試週的情況我常常在考試的時候都表現得不太好。此外，考試月也可以好好睡覺，比較不會有爆肝熬夜讀書的情形發生。

**Comment by yyujimmy** [comment]
兩種考試時程都有各自的優點和缺點，而喜歡哪一種可能取決於個人的學習風格和應對壓力的能力。

對於期末考週，優點在於一次性應對所有科目，考試壓力集中在一週之內，如果能夠有效規劃和準備，就可以在短時間內應對完畢，之後便能夠享受較長的假期。不過，這樣的集中考試也可能造成學習壓力過大，導致焦慮和疲憊。

而期末考月則是分散在一個月的時間內，每週只需應對一兩科目的考試，相對於期末考週來說，壓力分散，學習和準備時間相對較多，因此可以更加深入地學習和準備每一科目。然而，長時間的考試期間也可能使人感到疲累和煩躁，對於持續性的學習動力和壓力管理能力提出了更高的要求。

因此，喜歡期末考週還是期末考月，取決於個人對於壓力集中和分散的接受程度，以及對於學習和準備的自我規劃能力。无论是哪种模式，都可以透过有效的时间管理和压力缓解技巧，让自己在考试期间保持良好的状态。

**Comment by KH1126** [comment]
我比較偏向考試月，因為這樣考試比較分散。
我在大二時體驗過考試周的感覺，每週都要讀書壓力蠻大的。
並且考試月因為考試較分散的關係，可以在熬夜準備考試後，還可以多睡一些。

**Comment by tzurou** [comment]
我比較喜歡考試月，因為可以留給每個科目更多時間準備，也不會每天和時間賽跑熬夜通宵，可以更好的分配自己的時間，效率會提高，對自己的身體也比較好。

**Comment by jylin34** [comment]
我個人喜歡考試月，每科考完還有比較多的時間緩衝來繼續準備下一科，不用每天熬夜，或長時間處在高壓下。記得之前有一次一天考三科，從早考到晚，腦力直接耗盡。

**Comment by happy246810** [comment]
本人的考試時程是考試月，我覺得雖然考試月確實相對考試周較為輕鬆。但每一週都有考試也會導致你沒辦法有一定的放鬆時間，有點像是你一直被考試追著跑。所以我覺得兩者皆有它自己的好處與壞處。

**Comment by YOGA012** [comment]
比起所有考試全部集中在同一週，我更喜歡考試分散在一個月內，這樣對我來說壓力不會太大，而且對於我這種考前才在抱佛腳的人來說，一次抱一科比一次抱好幾科來的好，不過就會變成一直被考試追著跑，但其實久了也就習慣了。
